lib/routes/cdu/tzggcdunews.ts

```diff
@@ -0,0 +1,80 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+
+export const route: Route = {
+    path: '/tzggcdunews',
+    categories: ['university'],
+    example: '/cdu/tzggcdunews',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['news.cdu.edu.cn/'],
+        },
+    ],
+    name: '成都大学官网通知公告',
```

Comment:
```suggestion
    name: '通知公告',
```
No need to repeat namespace name in route name.